docs/source/models/causes/neonatal/index.rst

```diff
@@ -158,32 +174,62 @@ unit time.
     * - Symbol
       - Name
       - Definition
-    * - mr
-      - mortality risk
-      - The probability that a simulant who was born alive dies during the neonatal period
+    * - enn_mr
+      - mortality risk during the early neonatal period
+      - The probability that a simulant who was born alive dies during the first 7 days
+    * - lnn_mr
+      - mortality risk during the late neonatal period
+      - The probability that a simulant who was born alive dies between day 8 to 28 of life
 
 
 Data Tables
 +++++++++++
 
-The neonatal death model requires only the probability of death (aka "mortality risk") for use
-in the decision graph. This will be computed as
+The neonatal death model requires only the probability of death (aka "mortality risk") for the early and late neonatal time periods.  But computing this for an individual simulant is a bit complicated.  It will follow the pattern from the general mortality component in :code:`vivarium_public_health`, and work in rate space to make the math simpler.  The final step will be converting from rates to risks.
+
+.. math::
+    \begin{align*}
+    \text{mr}_i &= 1-\exp(-\text{ACMR}_i \cdot \Delta t),
+    \end{align*}
```

Comment:
If we compute the mortality risk this way, do you think it will match deaths per live birth as computed from GBD?

Reply:
I think it will, but Zeb just reminded us of an epic discussion about whether the `exp` is perhaps worse than a simpler version.